docs/ReleaseNotes.md

```diff
@@ -15,11 +15,26 @@ using 8-byte integers must now also explicitly declare such functions
 as 4-byte integers before calling them.
 [[Issue #78](https://github.com/NOAA-EMC/NCEPLIBS-bufr/issues/78)]
 
+* The interface between the C and Fortran components of the library has
+been modernized.  Accordingly, Fortran application codes must now contain a
+<b>use bufr_interface</b> statement to directly call any C functions within
+the library, and C application codes must now contain an
+<b>#include "bufr_interface.h"</b> statement to directly call any Fortran
```

Comment:
Here is your first doxygen problem. It generates this error:

/home/ed/NCEPLIBS-bufr/docs/ReleaseNotes.md:24: error: explicit link request to 'include' could not be resolved (warning treated as error, aborting now)

The problem is you have a hash sign before the include, so doxygen is thinking this is a doxygen command and not being able to cope.

When you want to use a hash in text, like this:

<b>\#include "bufr_interface.h"</b> statement to directly call any Fortran

Reply:
OK thanks.  Can you please let me know what functions don't have all the parameters defined?

Reply:
The backslash escape has been added, and I've also cleaned up the bufr_c2f_interface documentation as noted above.

But Doxygen is still not happy.  And again, I'm unable to reproduce this error on my system, even if I temporarily remove the -j2 switch.